src/lunchselector/db.clj

```diff
@@ -3,27 +3,170 @@
             [clj-time.core :as time]
             [clj-time.coerce :as coerce]))
 
-(def table "lunchvotes")
+(def votes "votes")
+(def users "users")
+(def restaurants "restaurants")
+
 (def db-path "localhost:5432/lunches")
 (def db-spec {:subprotocol "postgresql"
               :subname (str "//" db-path)})
 
-(defn insert [row]
-  (let [user (:user row)
+(defn add-user [user]
+  (let [name (:name user)
+        email (:email user)]
+    (jdbc/insert! db-spec
+                  (keyword users)
```

Comment:
For some reason it was not working for me. Maybe a typo. It works now. I have changed.